configs/whitelist.example.json

```diff
@@ -0,0 +1,3 @@
+{
+    "whitelist": ["jupyterhub:443"]
```

Comment:
Could we add this config into the config.example.json? Add a field like `jupyter_whitelist: []`

Reply:
Done !